pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -77,7 +77,7 @@ type Replacement struct {
 }
 
 func (c *Command) Reason() string {
```

Comment:
I'm fine with it for now, but calling this function reason with us using reason elsewhere is a bit odd